ReleaseNotes.md

```diff
@@ -33,6 +33,9 @@
     * Round tranformation can round to an integer or decimal value
   * `fluid.stringTemplate` updated to support nested objects for interpolating values
   * Added `fluid.dataSource` grade
+    * _**NOTE:** The scheme for configuring writable dataSources is not compatible with that seen in some development
+      releases of Infusion, e.g. 3.0.0-dev and 4.0.0-dev - see [API Changes documentation](https://docs.fluidproject.org/infusion/development/apichangesfrom2_0to3_0#datasources)
+      for more details_
```

Comment:
Would you be able to clarify this comment a bit more, or maybe something is missing from where the link goes to?